src/energyplus/ForwardTranslator/ForwardTranslateEnergyManagementSystemActuator.cpp

```diff
@@ -65,113 +69,203 @@ namespace openstudio {
-          if (result) {
-            m_idfObjects.push_back(result.get());
-          }
-          return result;
+      std::string zoneOrSpaceName = specificSpace_->nameString();
 
-        } else if (space) {
+      // Try to be helpful in case the user assigned a Space, but the FT doesn't translate spaces
+      if (m_forwardTranslatorOptions.excludeSpaceTranslation()) {
+        LOG(Warn, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                               << "' and specified that it wanted to be attached to the Space '" << specificSpace_->nameString()
+                               << "' but you have turned off ForwardTranslation's Space Feature. Falling back to using the attached ThermalZone '"
+                               << tz_->nameString() << "'.");
+        zoneOrSpaceName = tz_->nameString();
+      }
 
-          // if load is assigned to a single space instead of a space type then the name will not be changed in EnergyPlus
-          idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentUniqueName, m.nameString());
-          idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentType, modelObject.actuatedComponentType());
-          idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentControlType, modelObject.actuatedComponentControlType());
-          m_idfObjects.push_back(idfObject);
+      IdfObject idfObject = createRegisterAndNameIdfObject(openstudio::IddObjectType::EnergyManagementSystem_Actuator, modelObject);
+      const std::string spaceLoadNameForEMS = zoneOrSpaceName + " " + actuatedComponent_->nameString();
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentUniqueName, spaceLoadNameForEMS);
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentType, modelObject.actuatedComponentType());
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentControlType, modelObject.actuatedComponentControlType());
+      return idfObject;
+    }
 
-          return idfObject;
-        } else {
-          LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load.get().name().get()
-                                  << "' which is not associated with a ThermalZone, it will not be translated.");
-          return boost::none;
+    if (const boost::optional<ThermalZone> specificZone_ = modelObject.thermalZone()) {
+      auto spaces = specificZone_->spaces();
+      if (spaces.empty()) {
+        LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                                << "' and specified that it wanted to be attached to the ThermalZone '" << specificZone_->nameString()
+                                << "' but the Thermal Zone has no Spaces, so it will not be translated.");
+        return boost::none;
+      }
```

Comment:
User assigned a Thermal Zone. If it has no space, we have no business translating it. (the Thermal Zone will not be translated itself...no geometry or loads)